include/wavm/WAVMWasmModule.h

```diff
@@ -112,6 +114,10 @@ namespace wasm {
 
         int getDataOffsetFromGOT(const std::string &name);
 
+        U32 allocateThreadStack();
+
+        std::unique_ptr<PlatformThreadPool> &getPool();
```

Comment:
As the current thread pool stuff is all OMP-specific, can you make that clear in all the names? It will avoid confusion with the pthread support. This would be `getOMPThreadPool()` or something along those lines.